index.rst

```diff
@@ -10,7 +10,9 @@ Rules of Engagement for Accessing Data During the Embargo Period
 Embargo Restrictions:
 =====================
 
-Embargoed data types cannot leave the USDF during the embargo period. All Rubin USDF account holders will have full access to data in the embargo rack. Any data can reside transiently on batch or kubernetes processing nodes but must be deleted after use.
+Embargoed data types cannot leave the USDF during the embargo period. 
+All Rubin USDF account holders will have full access to data in the embargo rack (unless their Rubin role requires them to only hold standard LSST User permissions, as may be the case for some Rubin Community Scientists, for example). 
```

Comment:
If @leannep agrees ok

Reply:
Why would "some community scientists" have reduced/restricted  access to data? Why only community scientists? All current  community scientists are fully on-project and have a USDF account,  and so have the same rights as all  Rubin staff

Reply:
@leannep We can generalize this sentence to remove mention of Community Scientists if you prefer - I'll suggest a change. I just thought this role would be a good example for a Rubin staff member who didn't _need_ to have access to the embargo rack to do their work, and fact might even be better placed _outside the embargo rack_ with permissions the same as the LSST Users so that they can appreciate the challenges faced by those community members.  But there are other Rubin roles that will not require the staff in them to be able to access to the embargo rack - DRP pipeline scientists, DRP systematics scientists, etc etc.

Reply:
@leannep Regarding "standard LSST User permissions". I assume that when we have LSST Users at the USDF, because they have successfully proposed for resources there through the RAC, they will have permissions set such that they cannot access the embargo rack. By definition, the data in the embargo rack has not yet been released to the LSST Users, so they will have no need (or right) to access the embargo rack. If check whether RDM has any documentation yet on LSST User accounts at the USDF - if it doesn't, I'll reword to avoid using terminology that doesn't exist yet...